Core/System/Repository/ProductRepository.cs

```diff
@@ -62,11 +75,11 @@ public bool Save(Product product)
 
             if (product.Id > 0)
             {
-                query = "UPDATE dbjanmos.product SET name=@Name,description=@Description,category=@Category,metricUnit=@MetricUnit,metricValue=@MetricValue WHERE id=@Id;";
+                query = "UPDATE dbjanmos.product SET name=@Name,description=@Description,category=@Category,metricUnit=@MetricUnit,metricValue=@MetricValue,status=@Status WHERE id=@Id;";
```

Comment:
You added a parameter to hold the status value, but it will not receive any value from where it is being called in the ProductModal. This could cause an issue when the user attempts to update a deleted product in the future. Blank parameters default to null, and the status field in the backend defaults to 0. If the database engine doesn't recognize any valid value (null), it will default to its own preset value.

![image](https://github.com/blackMisay/Janmos/assets/39677820/51a43920-2305-497c-804c-58768a61253b)